include/stl2/view/iota.hpp

```diff
@@ -52,8 +52,7 @@ STL2_OPEN_NAMESPACE {
 			constexpr explicit iota_view(I value) requires Same<Bound, unreachable>
 			: value_(value), bound_{} {}
 			template <class II = I, class BB = Bound>
-			requires Same<I, II> && Same<Bound, BB>
-			constexpr iota_view(II value, BB bound)
+			constexpr iota_view(meta::id_t<II> value, meta::id_t<BB> bound)
```

Comment:
What the hell is this?

Reply:
A constructor with effectively no implicit deduction guide, so that `iota_view{0, std::size_t{1}}` fails class template argument deduction.

Reply:
*_*HATE.*_*

Reply:
Sold.  I now hate implicit deduction guides too.

Reply:
> Sold. I now hate implicit deduction guides too.

We *really* need the ability to `delete` guides.